roles/nova-data/tasks/v7k.yml

```diff
@@ -0,0 +1,42 @@
+---
+- block:
```

Comment:
you can remove the block and just have following tasks that will be smart to account operating system an install appropriate pkg and restart service.
i.e. in roles/openstack-meta/defaults/main.yml (under nova section add following after line 263)
```
      multipath:
        ubuntu:
          name: multipath-tools
          desc: multipath Service
        rhel:
          name: multipathd
          desc: multipath Service
```
in roles/nova-common/defaults/main.yml (define the multipath pkg name)
```
under nova.service add - multipath: "{{ openstack_meta.nova.services.multipath[ursula_os] }}"
also add following under nova.  (i.e u will use this as nova.multipath[os].package_name )
   multipath:
      ubuntu:
          package_name: multipath-tools
      rhel:
          package_name: device-mapper-multipath
```